src/main/java/org/jenkinsci/test/acceptance/FallbackConfig.java

```diff
@@ -182,9 +184,11 @@ private WebDriver createWebDriver(TestCleaner cleaner, TestName testName) throws
             if (StringUtils.isBlank(u)) {
                 throw new Error("remote-webdriver-firefox requires REMOTE_WEBDRIVER_URL to be set");
             }
+            DesiredCapabilities capabilitiesFirefox = DesiredCapabilities.firefox();
+            capabilitiesFirefox.setCapability("dom.disable_beforeunload", false);
```

Comment:
did you also want to use the static field here?

```suggestion
            capabilitiesFirefox.setCapability(DOM_DISABLE_BEFOREUNLOAD, false);
```